tools/triage/utils/filesystem/filesystem_helpers.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
Probably could do without this wrapper

Reply:
Will handle in a different PR

Reply:
I'm also puzzled as to why it's needed
Why won't we remove it as part of this PR? it's just copy-pasting the lines